src/pages/cart/Cart.jsx

```diff
@@ -0,0 +1,67 @@
+import React from "react";
+import "./cart.css";
+import { useDocumentTitle } from "../../utils";
+import { Card } from "../../components";
+import { useCart } from "../../context";
+import { Fa6SolidDove } from "../../assets/Icons";
+
+function Cart() {
+  useDocumentTitle("StarCart - Cart - Rohit Gaur");
+
+  const { cartProducts } = useCart();
+
+  const initialState = {
+    cartQuantity: 0,
+    totalPrice: 0,
+    delivery: 9,
+    discount: 0,
+  };
+
+  const checkout = cartProducts.reduce(
+    (acc, product) => ({
+      ...acc,
+      cartQuantity: acc.cartQuantity + product.cartQuantity,
+      totalPrice: acc.totalPrice + product.price.current * product.cartQuantity,
+    }),
+    initialState
+  );
+
+  return (
+    <main className="cartpage">
+      <div className="cart_products_area flex_column flex_gap2">
+        {cartProducts.map((product) => {
+          return (
+            <Card key={product.id} product={product} btnTitle="Move to cart" />
```

Comment:
You can convert this into single line

Reply:
thanks for pointing it out. updated

> The code is clean. Good use of all concepts together. One suggestion: To add build folder in git ignore (it is already added by default for me) as deploying platforms anyway run the build command and make a fresh build

Yes it was added for me as well. I removed it on purpose to build it at my end instead of the platform. Will add the filter approach so that they are excluded from PR perspective.